package com.zy.day4.exceptiom01;



public class ExceptioTest02 {

	public static void main(String[] args) {
		
		try {
			
		} catch (Exception e) {
			// TODO: handle exception
			e.printStackTrace();
		}
	}

}
